examples/mnist.py

```diff
@@ -23,15 +33,12 @@
 
 
 def _load_data(url=DATA_URL, filename=DATA_FILENAME):
-    try:
-        with gzip.open(filename, 'rb') as f:
-            data = pickle.load(f)
-    except (IOError, EOFError):
+    if not os.path.exists(filename):
```

Comment:
The `try..except` block had the advantage that it also caught an incomplete download.